src/components/v2/Table/index.tsx

```diff
@@ -29,6 +31,7 @@ export interface ITableProps {
   rowOnClick?: (e: React.MouseEvent<HTMLDivElement>, row: ITableRowProps[]) => void;
   className?: string;
   gridTemplateColumns?: string;
+  isMobileView?: boolean;
```

Comment:
This property is adjustable by parent because in different cases we need to switch to a mobile view of tables at different resolutions

Reply:
We need to use CSS for that. I made the change